examples/error-detail/error_detail.metatags

```diff
@@ -0,0 +1,2 @@
+description: This BBE demonstrates error detail in Ballerina.
```

Comment:
```suggestion
description: This BBE demonstrates error detail in Ballerina.
```
Is it `how to identify error details?`. The purpose of the BBE is not clear.

Reply:
Like in error cause we can also add details about the error. This shows how to do that

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-distribution/pull/3096/files/51b67d58f5b612e741782061ebdc043659093b81.